pkgs/development/ocaml-modules/res/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib , fetchFromGitHub , buildDunePackage }:
+
+buildDunePackage rec {
+  pname = "res";
+  version = "5.0.1";
+
+  minimalOCamlVersion = "4.08";
+
+  src = fetchFromGitHub {
```

Comment:
You may want to get the source from there instead: https://github.com/mmottl/res/releases/download/5.0.1/res-5.0.1.tbz

Reply:
Sure, sounds good -- I'll use `fetchurl` instead here.